server/services/owntracks/lib/handleMqttMessage.js

```diff
@@ -0,0 +1,46 @@
+/* eslint dot-notation: [2, {"allowPattern": "(_[a-z]+)+$"}] */
+
+const logger = require('../../../utils/logger');
+
+/**
+ * @description Handle a new message receive in MQTT.
+ * @param {string} topic - MQTT topic.
+ * @param {Object} message - The message sent.
+ * @returns {undefined}
+ * @example
+ * handleMqttMessage('owntracks/location/deviceId', data);
+ */
+function handleMqttMessage(topic, message) {
+  const splittedTopic = topic.split('/');
+  const user = splittedTopic[2];
+
+  if (topic.startsWith('owntracks')) {
+    logger.debug(`MQTT : Gladys has a message for ${user} from Owntracks on ${topic} :${message}.`);
+    const obj = JSON.parse(message);
+    if (obj['_type'] === 'location') {
+      // TODO : check is user exists or get user id using the selector
+      (async () => {
+        const users = await this.gladys.user.get({
+          selector: user,
+          take: 1,
+          skip: 0,
+        });
+
+        if (users) {
+          const data = {
+            user_id: users[0].id,
+            latitude: obj.lat,
+            longitude: obj.lon,
+            accuracy: obj.acc,
+            altitude: obj.alt,
+          };
+          await this.gladys.location.handleNewGatewayOwntracksLocation(data);
```

Comment:
I'm not sure calling the gladys Gateway code make sense, the Gladys Gateway is something very specific. 

Maybe now that we have an owntracks integration, we could have a function here that's called by the Gladys Gateway as well, and not the opposite ?